scripts/make-analysis-and-display.sh

```diff
@@ -59,7 +59,7 @@ tippecanoe \
 
 export MapboxAccessToken=$MAPBOX_ACCESS_TOKEN
 mapbox-upload \
-    "${MAPBOX_ACCOUNT}.vietnam-conflated" \
+    "${MAPBOX_ACCOUNT}.vietnam-conflated-1" \
```

Comment:
What's this "-1" for?

Reply:
@kamicut this PR is based off of master. We lost the .vietnam-conflated map id due to a processing bug which I need mapbox support to release. Until then we're using the -1 map id, the front-end already points to this.

Reply:
👍